config/initializers/secret_token.rb

```diff
@@ -1,4 +1,4 @@
-require_dependency 'foreman/util'
+require 'foreman/util'
 include Foreman::Util
```

Comment:
I'm creating RM ticket to remove this ugly global include, but `require` is correct here as it is in `lib` :+1:

Reply:
[RM#33977](https://projects.theforeman.org/issues/33977)